test/data/251.yml

```diff
@@ -0,0 +1,7 @@
+---
+:class: MemberPage
+:url: http://sitl.diputados.gob.mx/LXIII_leg/curricula.php?dipt=251
+:to_h:
+  :name: Héctor Ulises Cristopulos Ríos
+  :image: http://sitl.diputados.gob.mx/LXIII_leg/fotos_lxiiiconfondo/088_FOTO_CHICA.jpg
+  :email: 'hector.cristopulos@congreso.gob.mx '
```

Comment:
Yay. You caught a problem with the underlying code :)

That trailing space should really be removed from the email address. You can either add a comment here to say that that's a known problem (with a link to the Issue raised for it), or just fix it in this PR (making it a bug-fix rather than a refactoring)